Reply/app/src/main/java/com/materialstudies/reply/ui/compose/ComposeFragment.kt

```diff
@@ -69,13 +89,24 @@ class ComposeFragment : Fragment() {
             email = composeEmail
 
             composeEmail.nonUserAccountRecipients.forEach { addRecipientChip(it) }
+            composeEmail.nonUserAccountRecipients.forEach{  addRecipientsCard(it) }
+
+            binding.handlers = containerTranformHandler
+
+            binding.recipientAddIcon.setOnClickListener {
+                containerTransformRecipientChipToCard()
+            }
+            binding.recipientCardView.setOnClickListener {
+                containerTransformRecipientChipToCard()
```

Comment:
similar question here, shouldn't clicking the card do `containerTransformRecipientCardToChip`?